src/Common/Services/Mail/Aws/AwsMailProviderOptions.cs

```diff
@@ -0,0 +1,12 @@
+namespace Passwordless.Common.Services.Mail.Aws;
+
+public class AwsMailProviderOptions : BaseMailProviderOptions
+{
+    public const string Provider = "aws";
```

Comment:
`base.Name` does not appear to be set in the constructor, like in `FileMailProviderOptions`?